cfme/fixtures/cli.py

```diff
@@ -0,0 +1,90 @@
+from utils.version import get_stream
+from cfme.test_framework.sprout.client import SproutClient
+from utils.appliance import current_appliance
+from utils.conf import cfme_data, credentials
+from utils.log import logger
+import pytest
+from wait_for import wait_for
+from cfme.test_framework.sprout.client import SproutException
+
+
+@pytest.yield_fixture(scope="function")
+def dedicated_db_appliance(app_creds):
```

Comment:
I just tried this, it works for the fixture but sprout_client.destroy_pool(request_id) on appliance.py is called before the test is run against the dedicated_db_appliance.